src/ray/core_worker/common.h

```diff
@@ -181,6 +183,8 @@ struct PlacementGroupCreationOptions {
   const std::vector<std::unordered_map<std::string, double>> bundles;
   /// Whether to keep the placement group persistent after its creator dead.
   const bool is_detached = false;
+  /// The maximum fraction of CPU cores this placement group can take up on each node.
+  const double max_cpu_fraction_per_node = false;
```

Comment:
should this be 1.0 by default?

Reply:
Ah not sure how that snuck through, good catch!

Reply:
It seems only provides the fraction of CPU, why not provide the fraction of memory too?

Reply:
I think we are planning to make this API private for the short term, so we can only do for CPU for now. Let's discuss if this is the okay direction in the long term in the meeting. (and then we can include other things like GPU or memory)